docs/content/en/docs/design/config.md

```diff
@@ -74,15 +74,12 @@ relative to the current directory in `helm/project`.
 
 We generally recommend placing the configuration file in the root directory of the Skaffold project.
 
-## Configuration dependencies
-
-In addition to authoring pipelines in a Skaffold configuration file, we can also import pipelines from other existing configurations as dependencies. Skaffold manages all imported and defined pipelines in the same session. It also ensures all artifacts in a required configs are built prior to those in current config (provided the artifacts have dependencies defined); and all deploys in required configs are applied prior to those in current config.
+## Multiple configuration support
 
-Note that in imported configurations, files are resolved relative to the location of imported Skaffold configuration file.
-
-### Local config dependency
+A single `skaffold.yaml` file can define multiple skaffold configurations in the schema described above using the separator `---`. If these configuration objects have the `metadata.name` property then you can individually invoke a selection of configurations to run in any skaffold session.
```

Comment:
we should introduce this as a "module" since the flag is called that... namely a module is when you have a named skaffold config

Reply:
done